blocks/large-manual-promo-block/README.md

```diff
@@ -2,10 +2,6 @@
 
 _Block containing a large manual promo component. Please provide a 1-2 sentence description of what the block is and what it does._
```

Comment:
can you update this to not include placeholder info. Do you remember what we talked about was placeholder info? if you need help writing this, please ask a dev or a product person

Reply:
Updated README.